pkgs/os-specific/linux/sgx-sdk/samples.nix

```diff
@@ -12,7 +12,7 @@ let
     buildInputs = [
       sgx-sdk
     ];
-    enableParallelBuilding = true;
```

Comment:
nit: Was parallel building buggy?

Reply:
Yes, it failed randomly. Also, it's not too important to have it for the samples: they're tiny and build quickly.

Reply:
Please switch to `false` instead and maybe add a comment. There are efforts to build all packages eventually in parallel.

Reply:
That makes sense, thanks. Force-pushed [a new version](https://github.com/NixOS/nixpkgs/compare/124ad4ba0539f59ebaef0f1095912e8d928eb65a..4f7f8d0b2dce31038d1fde0d5d8321e68f16e0c6).